src/libxtp/factories/calculatorfactory.cc

```diff
@@ -19,6 +19,7 @@
 
 // Local VOTCA includes
 #include "votca/xtp/calculatorfactory.h"
+#include <iostream>
```

Comment:
?

Reply:
I always leave it hanging around :). I am going to remove it, thanks!

Reply:
is it needed?

Reply:
removed